crates/node/src/chain_spec.rs

```diff
@@ -85,9 +88,27 @@ pub fn development_config(enable_manual_seal: Option<bool>, madara_path: PathBuf
     ))
 }
 
+// helper to print development accounts info
+// accounts with addresses 0x1 and 0x4 are NO VALIDATE accounts (don't require PK)
+// accounts with addresses 0x2 and 0x3 have the same PK
+pub fn print_development_accounts(genesis_loader: GenesisLoader) {
+    let genesis_config: madara_runtime::pallet_starknet::GenesisConfig<Runtime> = genesis_loader.into();
+    let no_validate_account_address = genesis_config.contracts[0].0.0.to_string();
+    let argent_account_address = genesis_config.contracts[1].0.0.to_string();
+    let oz_account_address = genesis_config.contracts[2].0.0.to_string();
+    let cairo_1_no_validate_account_address = genesis_config.contracts[3].0.0.to_string();
+
+    const ARGENT_PK: &str = "0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d";
+    log::info!("🧪 Using the following development accounts:");
+    log::info!("🧪 NO VALIDATE with address: 0x{} and no pk", no_validate_account_address);
+    log::info!("🧪 ARGENT with address: 0x{} and pk: {}", argent_account_address, ARGENT_PK);
+    log::info!("🧪 OZ with address: 0x{} and pk: {}", oz_account_address, ARGENT_PK);
+    log::info!("🧪 CAIRO 1 with address: 0x{} and no pk", cairo_1_no_validate_account_address);
+}
```

Comment:
Rather than using `to_string` and manually prefixing by `0x{}`, you can use rust native https://doc.rust-lang.org/std/fmt/trait.LowerHex.html.

Implement it on your `HexFelt` type and enjoy

https://stackoverflow.com/questions/56485167/how-to-format-a-byte-into-a-2-digit-hex-string-in-rust

Reply:
Good idea, will do it this week.

Reply:
Done